frontier-chain/precompile/living-assets/contracts/LivingAssetsEvolution.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+/// @title Pallet Collection Manager Interface
+/// @author Freeverse team
+/// @notice This interface allows Solidity contracts to interact with pallet-living-assets
+/// @custom:address 0x0000000000000000000000000000000000000402
+interface LivingAssets {
+    /// @notice Event emitted when a new collection is created
+    /// @param collectionId Id of the newly created ERC721 collection
+    /// @param owner Owner of the newly created ERC721 collection
+    event CreateCollection(uint256 collectionId, address indexed owner);
+
+    /// @notice Creates a new collection
+    /// @dev Call this function to create a new collection
+    /// @return address of the ERC721 collection
+    function createCollection() external returns (address);
```

Comment:
Recall that createCollection must accept an owner as input param. In other words, the sender is not necessarily the owner.

Reply:
ah, ok, makes sense